settings/src/components/auto-tabbing-input.js

```diff
@@ -11,26 +11,34 @@ import NumericControl from './numeric-control';
 const AutoTabbingInput = ( props ) => {
 	const { inputs, setInputs, error, setError } = props;
 
-	const handleChange = useCallback( ( value, event, index, inputRef ) => {
+	const handleChange = useCallback( ( value, event, index ) => {
 		setInputs( ( prevInputs ) => {
 			const newInputs = [ ...prevInputs ];
 
 			newInputs[ index ] = value.trim() === '' ? '' : value;
 
 			return newInputs;
 		} );
+	}, [] );
 
-		if ( value && '' !== value.trim() && inputRef.current.nextElementSibling ) {
-			inputRef.current.nextElementSibling.focus();
+	const handleKeyUp = useCallback( ( value, event, index, inputElement ) => {
```

Comment:
Handling auto focus of the next/prev input `onKeyUp` means we have the new value, but aren't relying on whether the value has changed to determine whether to shift focus.